Beginners/Lesson Two/README.md

```diff
@@ -0,0 +1,153 @@
+Alright, lesson 2!
+
+### Overview of last week
+
+vars/if/for/print
+
+So the main topic we're going to cover today is logic and then we'll look at types.
+
+### Bools
+
+In the last lesson we used a lot of variables. A variable is a way in which we could label something that can change. The variables we used were numbers (`Int`s) or collections of characters (`String`s). What about when a variable should only be a certain type of data. Let's look at an on/off value (`Bool`). These are pretty easy to work with, they can either be `true` or `false`. Lets make some.
+
+``` swift
+var isThisExcitingYet = false
+```
+
+Alright cynic. Let's do something more interesting.
+
+``` swift
+var isThisExcitingYet = false
+var areYouMean = true
+```
+
+### Iffy Logic
+
+Ok, we have two variables. These work really nicely with `if` statements.
+
+``` swift
+var isThisExcitingYet = false
+var areYouMean = true
+
+if isThisExcitingYet {
+  print("🎉")
+}
+```
+
+This will check if it's fun yet, and if so Xcode will print a party popper. We could cheat and instead check if it's false, then we get a free party popper.
+
+``` swift
+if isThisExcitingYet == false {
+  print("🎉")
+}
+```
+
+However, we can do it right. `if` statements can be powerful. We've been using only the simplest part so far. We're going to look at `else`, which bolts on to an `if` and provides a way of running some code when the `if` fails.
+
+``` swift
+# OK, we have a way to protect against boredom.
+
+if isThisExcitingYet {
+  print("🎉")
+} else {
+  print("Please visit https://www.youtube.com/results?search_query=andrew%20wk")
+}
+```
+
+So what we're looking at, in plain English: _if something is exciting, show a party popper else recommend a good youtube video_. We can use an `if` and an ` else` statement to represent a very large amount of logic. `if`s have one more thing, that I'd like to go over. Before we do that, we should consider what happens if Andrew WK is in the room.
+
+``` swift
+var isAndrewWKAround = false
+```
+
+Well, he would have an important impact on everyone's exciting-ness. Let's try and represent that in our if statement.
+
+``` swift
+# Whoah, Andrew WK?
```

Comment:
Given the large amount of "repeating" over the same code path, I wanted to be able to ensure there is an easy way to differentiate in the playground as will show a lot of repeated code bits